### PR TITLE
Support different API versions of ingress

### LINKED ISSUE #17: Support newer Ingress apiVersion
Currently ingresses are created with apiVersion `extensions/v1beta1`, which is already deprecated and will be removed in couple of versions.

ApiVersion should be set with logic, that if version is higher than e.g. 1.19, ingress should be created with apiVersion `networking.k8s.io/v1beta1`.

This would affect to existing deployments, too, as cert-manager `Certificate` object owner is Ingress.

### PR DESCRIPTION
**Pull request by Hi-Fi**
Fixes #17

# Does this PR meet the acceptance criteria

* **Changelog**
    * [x] Changelog entry included
    * [ ] Changelog entry not needed
* **Documentation**
    * [ ] Documentation included
    * [x] Documentation not needed
* **Tests**
    * [ ] Tests included
    * [x] Tests not needed

### CODE REVIEW

**Comment by Hi-Fi** [comment]
@frwickst How about this? Without the change update to Kubernetes 1.22 is not possible (that would be available e.g. in Azure at October and in GCP at August). Of course 1.21 EOL would be around next year summer, but it would be a bit hassle to have to update multiple versions at once if change comes later.

File: helm/templates/05-ingress.yaml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {{- if and (not .Values.ingress.disabled) (not .Values.jobsOnly) -}}
+{{- $stableApi := .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress"}}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" -}}
```
Comment (frwickst) [inline helm/templates/05-ingress.yaml]: Could this be just `{{- if $stableApi -}}` ?

**Review by frwickst — CHANGES_REQUESTED** [review]
Thank you @Hi-Fi !
I totally missed this one, I'm really sorry. Need to check that our Slack integration is properly set up in the future so that i don't miss your contributions :D

File: helm/templates/05-ingress.yaml (hunk 1 above)
Comment (Hi-Fi) [reply helm/templates/05-ingress.yaml]: I think yes, can't remember at least directly if there was some issues with the variable.

**Pull request merged**